### PR TITLE
Allow commands not to be signed

### PR DESCRIPTION
**Pull request by forester302**
Updated chat.js as suggested in https://github.com/PrismarineJS/node-minecraft-protocol/issues/1262

### CODE REVIEW

**Review by extremeheat — APPROVED** [review]
I don't like that the chat function has so much duplicative code there, it's rather hard to follow. It could do with some simplification. Also `_session` is a bad variable name, it's not clear to me what it has to do with chat.

But anyway LGTM for a temporary fix, cc @frej4189

**Comment by frej4189** [comment]
LGTM.

I don't think this is a temporary fix. This is exactly how it's supposed to work
